ow_lander/scripts/trajectory_publisher.py

```diff
@@ -131,24 +133,50 @@ def talker(req):
       print "Sent %s on joint[0] publisher"%(float("%14s\n"%row[12+0]))
       rate.sleep()
 
-        
+
   if guarded_move_bool : # If the activity is a guarded_move
+
+    # Start publisher for guarded_move result message
+    pubs.append(rospy.Publisher('/guarded_move_result', GuardedMoveResult, queue_size=10))
+
     # Start subscriber for the joint_states
     rate = rospy.Rate(int(pub_rate/2)) # Hz
     rospy.Subscriber("/joint_states", JointState, joint_states_cb)
     time.sleep(2)
-    start_guard_delay_acc = 0 
+    start_guard_delay_acc = 0
     for row in guard_rows[1:]: # Cycles on all the rows except header
       if row[0][0] == '1' : # If the row is a command
-        ground = check_for_contact(velocity_array)  
-        if (ground == 1):     
+        ground = check_for_contact(velocity_array)
+        if (ground == 1):
             print "Found ground, stopped motion..."
+
+            listener = tf.TransformListener()
+            listener.waitForTransform("/base_link", "/l_scoop_tip", rospy.Time(0), rospy.Duration(10.0))
+            (trans,rot) = listener.lookupTransform("base_link", "l_scoop_tip", rospy.Time(0))
+            pubs[6].publish(ground, trans[0], trans[1], trans[2], 'base_link')
+
+            # msg = GuardedMoveResult()
+            # msg.success = ground
+            # msg.X = trans[0]
+            # msg.Y = trans[1]
+            # msg.Z = trans[2]
+            # msg.frame = 'base_link'
+            # print(trans)
+            # print(msg)
+            # print(GuardedMoveResult)
```

Comment:
Is this for your own debugging? If so no need to push to it, git allows you to commit and parts of the file.